starknet_py/tests/e2e/docs/account_creation/test_deploy_prefunded_account.py

```diff
@@ -43,26 +43,21 @@ async def test_deploy_prefunded_account(
     await res.wait_for_acceptance()
     # docs: start
 
-    # Create an Account instance
-    account = Account(
+    # Use `Account.deploy_account` static method to deploy an account
+    account_deployment_result = await Account.deploy_account(
         address=address,
-        client=GatewayClient(net=network),
+        class_hash=class_hash,
+        salt=salt,
         key_pair=key_pair,
+        client=GatewayClient(net=network),
         chain=StarknetChainId.TESTNET,
```

Comment:
Shouldn't we maybe use here (and possibly in other places in documentation) `chain_from_network()`? It's currently hardcoded to `StarknetChainId.TESTNET`, but the chain should be dependent on the network, as it's not specified above to be `testnet`.

Reply:
I would rather change the network to `TESTNET`. Our examples are based on testnet

Reply:
But shouldn't the examples be as universal as possible? The network might as well be `TESTNET2`.

Reply:
Right, but if someone sees how to use TESTNET it can use TESTNET2 as well. With the `network`, we would have to show what it really is